`[SEVERE] : Skipped generating bindings due to errors in source files. See https://github.com/dart-lang/native/blob/main/pkgs/ffigen/doc/errors.md.`

https://github.com/sensuikan1973/libedax4dart/actions/runs/7426658505

```
2024-01-03T21:02:50.4491510Z [SEVERE] :     ./edax-reversi/src/obftest.c:484:46: error: unknown type name 'BenchResult' [Semantic Issue]
```

okok

--- a/ffigen.yaml
+++ b/ffigen.yaml
@@ -4,6 +4,7 @@ name: 'LibEdaxBindings'
 description: 'bindings to libedax'
 output: 'lib/src/ffi/bindings.dart'
 sort: true
+ignore-source-errors: true # workaround for https://github.com/sensuikan1973/libedax4dart/issues/482
 # See: https://github.com/dart-lang/pana/issues/1013
 # See: https://dart.dev/guides/language/analysis-options#excluding-code-from-analysis
 preamble: "// ignore_for_file: type=lint, unused_element, unused_field"

--- a/lib/src/ffi/bindings.dart
+++ b/lib/src/ffi/bindings.dart
@@ -37,8 +37,6 @@ class LibEdaxBindings {
 
   int get BOOK_INFO_RESOLUTION => _BOOK_INFO_RESOLUTION.value;
 
-  set BOOK_INFO_RESOLUTION(int value) => _BOOK_INFO_RESOLUTION.value = value;
-
   late final ffi.Pointer<ffi.Pointer<ffi.Char>> _COUNT_FLIP_2 =
       _lookup<ffi.Pointer<ffi.Char>>('COUNT_FLIP_2');
 
@@ -174,22 +172,16 @@ class LibEdaxBindings {
 
   int get EVAL_N_FEATURE => _EVAL_N_FEATURE.value;
 
-  set EVAL_N_FEATURE(int value) => _EVAL_N_FEATURE.value = value;
-
   /// number of plies
   late final ffi.Pointer<ffi.Int> _EVAL_N_PLY = _lookup<ffi.Int>('EVAL_N_PLY');
 
   int get EVAL_N_PLY => _EVAL_N_PLY.value;
-
-  set EVAL_N_PLY(int value) => _EVAL_N_PLY.value = value;
 
   /// number of (unpacked) weights
   late final ffi.Pointer<ffi.Int> _EVAL_N_WEIGHT =
       _lookup<ffi.Int>('EVAL_N_WEIGHT');
 
   int get EVAL_N_WEIGHT => _EVAL_N_WEIGHT.value;
-
-  set EVAL_N_WEIGHT(int value) => _EVAL_N_WEIGHT.value = value;
 
   /// feature offset
   late final ffi.Pointer<ffi.Pointer<ffi.Int>> _EVAL_OFFSET =
@@ -441,8 +433,6 @@ class LibEdaxBindings {
       _lookup<ffi.Int>('NO_SELECTIVITY');
 
   int get NO_SELECTIVITY => _NO_SELECTIVITY.value;
-
-  set NO_SELECTIVITY(int value) => _NO_SELECTIVITY.value = value;
 
   /// threshold values to try stability cutoff during NWS search
   late final ffi.Pointer<ffi.Pointer<ffi.Int>> _NWS_STABILITY_THRESHOLD =
@@ -1528,8 +1518,6 @@ class LibEdaxBindings {
   late final ffi.Pointer<ffi.Int> _sys_nerr = _lookup<ffi.Int>('sys_nerr');
 
   int get sys_nerr => _sys_nerr.value;
-
-  set sys_nerr(int value) => _sys_nerr.value = value;
 
   late final ffi.Pointer<ffi.Pointer<ffi.Pointer<ffi.Char>>> _sys_siglist =
       _lookup<ffi.Pointer<ffi.Pointer<ffi.Char>>>('sys_siglist');

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -19,7 +19,7 @@ dev_dependencies:
   analyzer: ^6.3.0
   cider: ^0.2.4
   coverage: ^1.7.1 # See: https://github.com/dart-lang/test/issues/1265
-  ffigen: ^10.0.0
+  ffigen: ^11.0.0
   pana: ^0.21.39
   pedantic_sensuikan1973: ^5.3.0
   test: ^1.24.9

